tasks/main.yml

```diff
@@ -122,6 +131,26 @@
   with_items:
     - pacemaker
     - corosync
+  when: not cluster_is_remote | bool
+
+- name: Get cluster status
+  command: pcs cluster status
+  run_once: true
+  changed_when: false
+  register: cluster_status
+
+- name: Add remote node
+  vars:
+    cluster_hostname: "{{ hostvars[inventory_hostname][cluster_hostname_fact] }}"
+    delegate_host: "{{ hostvars[groups['cluster_is_remote_False'][0]].inventory_hostname }}"
+    # NOTE: Without this, the host's ansible_host variable will not be
+    # respected when using delegate_to.
+    ansible_host: "{{ hostvars[delegate_host].ansible_host | default(delegate_host) }}"
+  command: pcs cluster node add-remote {{ cluster_hostname }}
```

Comment:
- [ ] `pcs cluster node add-remote` works well for me in CentOS 7.8 and CentOS 8.2, not in CentOS 6.10 (syntax seems to be different). If your main use case is to have this on CentOS 8.x I propose to limit this only on EL7 and EL8.
For reference the CentOS 6.10 syntax, it looks to me that it needs already configured remote resource to work properly adding node. I don't mind if the CentOS 6.10 is not supported here.
~~~
# pcs cluster remote-node add --help

Usage: pcs cluster remote-node...
    remote-node add <hostname> <resource id> [options]
        Enables the specified resource as a remote-node resource on the
        specified hostname (hostname should be the same as 'uname -n').

    remote-node remove <hostname>
        Disables any resources configured to be remote-node resource on the
        specified hostname (hostname should be the same as 'uname -n').
~~~

Reply:
Making this a 7+ feature works for me.